### PR TITLE
Release nitclk 1.0, nidigital 0.5, and 1.2 of other modules

### PR DESCRIPTION
**Pull request by sbethur**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

~- [ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?

Release the following:
- nitclk 1.0
- nidigital 0.5
- nidcpower 1.2
- nidmm 1.2
- nifgen 1.2
- nimodinst 1.2
- niscope 1.2
- nise 1.2
- niswitch 1.2

### List issues fixed by this Pull Request below, if any.

None

### What testing has been done?

Unit and system tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
```
Comment (marcoskirsch) [inline CHANGELOG.md]: you should add the placeholder for the next [Unreleased]
or is the plan to add it in a follow up PR?

File: README.rst [hunk 2]
```diff
@@ -70,7 +70,7 @@ NI-DCPower Python API Status
 +-------------------------------+--------------------------+
 | NI-DCPower (nidcpower)        |                          |
 +===============================+==========================+
-| Driver Version Tested Against | 19.1.0                   |
+| Driver Version Tested Against | 19.5.0                   |
```
Comment (marcoskirsch) [inline README.rst]: This change should have been made when nimi-bot was upgraded. Opened #1314.

File: README.rst [hunk 3]
```diff
@@ -148,7 +148,7 @@ NI-DMM Python API Status
 +-------------------------------+----------------------+
 | NI-DMM (nidmm)                |                      |
 +===============================+======================+
-| Driver Version Tested Against | 19.1.0               |
+| Driver Version Tested Against | 19.5.0               |
```
Comment (marcoskirsch) [inline README.rst]: This is the version of the NI-DMM full installation, but in some cases the runtime alone is not revved up and may be older.

I think we need to list the runtime version here even if older than the "full SDK installation" version.

File: generated/nidigital/setup.py [hunk 4]
```diff
@@ -51,7 +51,7 @@ def read_contents(file_to_read):
     tests_require=['pytest'],
     test_suite='tests',
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```
Comment (marcoskirsch) [inline generated/nidigital/setup.py]: I think we should keep this in Alpha still. Beta once we think the API is stable.

File: CHANGELOG.md (hunk 1 above)
Comment (sbethur) [reply CHANGELOG.md]: Yes, there'll be a follow-up PR called "Post Release"

File: README.rst (hunk 3 above)
Comment (sbethur) [reply README.rst]: Fixed

File: generated/nidigital/setup.py (hunk 4 above)
Comment (sbethur) [reply generated/nidigital/setup.py]: Fixed

File: docs/niswitch/examples.rst [hunk 5]
```diff
@@ -1,7 +1,7 @@
 Examples
 ========
 
-`You can download all niswitch examples here <https://github.com/ni/nimi-python/releases/download/1.1.5/niswitch_examples.zip>`_
+`You can download all niswitch examples here <https://github.com/ni/nimi-python/releases/download/1.2.0/niswitch_examples.zip>`_
```
Comment (marcoskirsch) [inline docs/niswitch/examples.rst]: we'll have to test this link once the artifacts are published.

**Pull request merged**